homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -152,7 +152,7 @@ def __init__(
     ) -> None:
         """Initialize the device."""
         super().__init__(device_url, coordinator)
-        self._attr_name = f"{self._attr_name} Low Speed"
+        self._attr_name = "Low speed"
```

Comment:
How does this change fixed https://github.com/home-assistant/core/issues/79237?

Reply:
In https://github.com/home-assistant/core/pull/76687 we removed `self._attr_name = self.device.label` from the OverkizEntity. The `_attr_name` is not set anymore, thus we cannot include it in the string here. (and we should not, since we set the has new entity naming to true.